curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-the-date-object-by-building-a-date-formatter/646ae02fc520da01e52e0d1e.md

```diff
@@ -0,0 +1,49 @@
+---
+id: 646ae02fc520da01e52e0d1e
+title: Step 5
+challengeType: 0
+dashedName: step-5
+---
+
+# --description--
+
+Create a `div` HTML element with the class name `title-container`
```

Comment:
Let's remove this step since this is the fourth project in the curriculum. 

By this point, we are supplying the campers with all of the HTML and CSS.

Reply:
removed